From 0bb52df211b51b96594f7e3bb20cef939c59b3c2 Mon Sep 17 00:00:00 2001
From: Valentin Caron <valentin.caron@foss.st.com>
Date: Mon, 25 Mar 2024 11:36:33 +0100
Subject: [PATCH] arm64: dts: st: add SCMI power domain in stm32mp251.dtsi

Add protocol 17 in SCMI node, to send power-domains change state
through SCMI protocol.

Signed-off-by: Valentin Caron <valentin.caron@foss.st.com>
Change-Id: I36f1e24771c5593291d33b19bd7eb65636ea61c5
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/373466
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/st/stm32mp251.dtsi b/arch/arm64/boot/dts/st/stm32mp251.dtsi
index 2c01ba65520a..427d34a48996 100644
--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -106,6 +106,11 @@ scmi {
 			#size-cells = <0>;
 			linaro,optee-channel-id = <0>;
 
+			scmi_devpd: protocol@11 {
+				reg = <0x11>;
+				#power-domain-cells = <1>;
+			};
+
 			scmi_perf: protocol@13 {
 				reg = <0x13>;
 				#clock-cells = <1>;
-- 
2.39.5

